packages/cryptoplease_link/public/solana-pay-styles.css

```diff
@@ -0,0 +1,220 @@
+@media screen and (min-width: 600px) {
+  body {
+    text-align: center;
+    padding: 10px;
+    position: relative;
+    background: #424242;
+    color: white;
+    overflow-wrap: break-word;
+    font-family: 'Roboto', sans-serif;
+    text-align: center;
+    letter-spacing: 0.007em;
+  }
+
+  .logo {
+    max-width: 100px;
+    margin: 32px auto 32px;
+    display: block;
+  }
+
+  .background {
+    z-index: -1;
+    height: 100%;
+    max-width: 100%;
+    display: block;
+    position: absolute;
+    margin: auto;
+    float: left;
+    left: -240px;
+    top: 8%;
+  }
+
+  .message-block {
+    z-index: 0;
+    display: block;
+    position: relative;
+    overflow: hidden;
+    max-width: 85%;
+    max-height: 45%;
+    background: linear-gradient(217.94deg, #f45440 10.03%, #ff8b49 100.98%);
+    border-radius: 35px;
+    margin: auto;
+    padding-top: 60px;
+    padding-bottom: 150px;
+  }
+
+  .payment-block {
+    font-size: 27px;
+    font-weight: 600;
+    margin: 32px auto 32px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .address {
+    background: #f4f4f4;
+    border: 1px solid #ededed;
+    border-radius: 12px;
+    overflow-wrap: break-word;
+    margin: 32px 32px;
+    display: block;
+    padding: 20px;
+    min-height: 56px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 21px;
+    line-height: 25px;
+    text-align: center;
+    color: #a7a7a7;
+  }
+}
+
+@media screen and (max-width: 600px) {
+  body {
+    text-align: center;
+    align-items: center;
+    align-self: center;
+    background: #f45440;
+    color: white;
+    overflow-wrap: break-word;
+    font-family: 'Roboto', sans-serif;
+    text-align: center;
+    letter-spacing: 0.007em;
+    background-image: url('solana-background-mobile.svg');
+    background-size: 110% auto;
+    background-repeat: no-repeat;
+    background-position: top;
+    height: 100%;
+    width: 100%;
+    display: block;
+    position: absolute;
+    margin: auto;
+    float: top;
+    top: 0;
+  }
+
+  .logo {
+    max-width: 100px;
+    margin: 64px auto 72px;
+    display: block;
+  }
+
+  .background {
+    height: 0;
+    max-width: 0;
+  }
+
+  .message-block-mobile {
+    margin: auto;
+    position: relative;
+    display: block;
+    top: 150;
+  }
+
+  .payment-block {
+    font-size: 27px;
+    font-weight: 600;
+    margin: 32px auto 32px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    float: bottom;
+    bottom: 0;
+  }
+
+  @media screen and (prefers-color-scheme: dark) {
+    .background-mobile {
+      position: relative;
+      display: block;
+      width: 100%;
+      height: 100%;
```

Comment:
If content fits the page, there should be no scroll:

<img width="488" alt="image" src="https://user-images.githubusercontent.com/853356/172711422-37ddaa72-2fd3-4ab3-85c2-e052ad6058cc.png">